### PR TITLE
Remove yarn refs in docs

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Remove yarn reference in docs. We don't support yarn first-class.

### CODE REVIEW

File: packages/terra-alert/docs/README.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ The Terra Alert component is a notification banner that can be rendered in your
 
 - Install with [npmjs][3]:
   - `npm install terra-alert`
-  - `yarn add terra-alert`
```
Comment (StephenEsser) [inline packages/terra-alert/docs/README.md]: Are these auto-generated when creating a new package?

Comment (emilyrohrbough) [reply packages/terra-alert/docs/README.md]: At one point, there were auto-generated, but we've made changes to remove this for new packages: https://github.com/cerner/generator-terra-module/pull/19/files#diff-4b8982f7ad1a52f85e37add894537d2fL20

**Comment by emilyrohrbough** [comment]
I'm unable to reproduce the terra-search-field wdio error locally. I've clean:installed multiple times and have tried these runs:
```
> npm run wdio-default
> npm run wdio-default -- --spec packages/terra-search-field/tests/wdio/search-field-spec.js
> npm run wdio-default -- --spec packages/terra-search-field/tests/wdio/search-field-spec.js
> FORM_FACTOR='medium' npm run wdio-default -- --spec packages/terra-search-field/tests/wdio/search-field-spec.js

```

It appears to pass on Master fine, however, this PR #2792 is also seeing this failure. Any one have ideas?

**Comment by yuderekyu** [comment]
@emilyrohrbough I re-ran tests for https://github.com/cerner/terra-core/pull/2792 today, looks like the search field passed. I'm not sure what changed.

**Comment by emilyrohrbough** [comment]
@yuderekyu Hopefully it just resolves itself?? 😬 I just ran the docker-compose setup used by travis on my machine and everything passed for me as well.. 😕

**Pull request merged**